hardware/BleBox.cpp

```diff
@@ -168,7 +200,7 @@ std::string BleBox::IPToHex(const std::string &IPAddress)
 	std::vector<std::string> strarray;
 	boost::split(strarray, IPAddress, boost::is_any_of("."));
 	if (strarray.size() != 4)
-		return "0.0.0.0";
+		return "";
```

Comment:
It seems your not handling this further in the code ? Is that correct? (its inserted empty in the database, should the insert take place?)